### ISSUE TITLE
Cloudflare support

### ISSUE THREAD

**Issue by 09173732546** [jgmdev/stopspam#1]
Hi,

Is Cloudflare CF-Connecting-IP enabled

**Comment by jgmdev** [comment]
No, and thanks for pointing it out will port ddos-deflate implementation into this later.
